### ISSUE TITLE
Inheritance in Embeddables

### ISSUE THREAD

**Issue by bartlomiej-korpus** [typeorm/typeorm#966]
```
class A {
  public a: string;
  public b: string;
}

class B extends A {
  public c: string;
}

@Entity()
class C {
@Column(type=>B)
public field: B;
}
```

Now, the auto schema sync only creates "c" column, so I guess thats the only one recognized by typeorm.

I will try to look deeper into this bug. It would be very helpful if someone could point me in the right direction, I am not very familiar with the codebase nor with reflect-metadata unfortunately.

**Comment by pleerock** [comment]
Would be great if you provide a failing test case first, which reproduce this issue.

**Comment by bartlomiej-korpus** [comment]
@pleerock I've sent a PR with a simplest test case I could think of that demonstrates the issue.

https://github.com/typeorm/typeorm/pull/980

**Comment by pleerock** [comment]
thanks I'll check it

**Comment by pleerock** [comment]
thank you for your test, it was really easy to fix this issue with it. Fix will be released in `0.1.1`.

**Comment by bartlomiej-korpus** [comment]
@pleerock Thank you! Awesome!
